lab_8_llm/start.py

```diff
@@ -1,16 +1,60 @@
 """
-Neural machine translation starter.
+Neural Classification starter.
 """
 # pylint: disable= too-many-locals
+from pathlib import Path
+
+from config.lab_settings import LabSettings
 from core_utils.llm.time_decorator import report_time
+from lab_7_llm.main import (LLMPipeline, RawDataImporter, RawDataPreprocessor, TaskDataset,
+                            TaskEvaluator)
 
 
 @report_time
 def main() -> None:
     """
-    Run the translation pipeline.
+    Run the classification pipeline.
     """
-    result = None
+    root_path = Path(__file__).parent
+    settings_path = LabSettings(root_path / "settings.json")
+    predictions_path = root_path / "dist" / "predictions.csv"
+
+    if not predictions_path.parent.exists():
+        predictions_path.parent.mkdir(exist_ok=True)
+
+    # mark 4
+    importer = RawDataImporter(settings_path.parameters.dataset)
+    importer.obtain()
+
+    # mark6
+    preprocessor = RawDataPreprocessor(importer.raw_data)
+    preprocessor.analyze()
+    preprocessor.transform()
+
+    dataset = TaskDataset(preprocessor.data.head(100))
+
+    batch_size = 1
+    pipeline = LLMPipeline(model_name=settings_path.parameters.model,
+                           dataset=dataset,
+                           max_length=120,
+                           batch_size=batch_size,
+                           device="cpu")
+    pipeline.analyze_model()
+    pipeline.infer_sample(next(iter(dataset)))
+
+    # mark 8
+    batch_size = 64
+    pipeline = LLMPipeline(model_name=settings_path.parameters.model,
+                           dataset=dataset,
+                           max_length=120,
```

Comment:
If the `batch_size` is in a separate variable, then why is the `max_length` not?

Reply:
corrected